extension/association_tools.js

```diff
@@ -116,9 +114,12 @@ function createAssociationBox({comment, linkToSE, SE_Question, absTime, SE_SiteN
         e.preventDefault();
     });
 }
-function questionId(uri) {
-    var id = /\/(\d+)\//.exec(uri)
-    return id == null ? -1 : id[1];
+function questionId(uri, site) {
```

Comment:
But, **site** argument value can be  parsed from **uri** content and looks like redundant.   

I was adding it just because we already parse it in **getSEQuestionId**